data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/config/CwlSinkConfig.java

```diff
@@ -0,0 +1,54 @@
+package org.opensearch.dataprepper.plugins.sink.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import org.opensearch.dataprepper.plugins.sink.configuration.AwsAuthenticationOptions;
+
+public class CwlSinkConfig {
+    public static final String DEFAULT_BUFFER_TYPE = "in_memory";
+
+    //Class was utilized from the
+    @JsonProperty("aws")
+    @NotNull
+    @Valid
+    private AwsAuthenticationOptions awsConfig;
```

Comment:
Let's do `private AwsConfig awsConfig;`? 
Duplicate `AwsAuthenticationOptions` code if needed.
We are working on moving this to a common location. For now, it's ok if you duplicate it.

Reply:
I have changed it to AwsConfig.